Makefile

```diff
@@ -1,20 +1,6 @@
-CUR_VER := $(shell poetry run ./current_version.py)
+CUR_VER ?= $(shell poetry run ./current_version.py)
```

Comment:
:= always runs the command
?= only runs the command if CUR_VER wasn't set as a bash or environment variable before calling `make`